phpBB/styles/prosilver/template/captcha_recaptcha.html

```diff
@@ -1,30 +1,48 @@
-<!-- IF S_TYPE == 1 -->
-<div class="panel captcha-panel">
-	<div class="inner">
+<!-- IF S_RECAPTCHA_AVAILABLE -->
+	<noscript>
+		<div>{L_RECAPTCHA_NOSCRIPT}</div>
+	</noscript>
+	<input id="recaptcha_helper" class="hidden">
+	<script src="{RECAPTCHA_SERVER}.js?onload=recaptcha_callback&hl={LA_RECAPTCHA_LANG}" async defer></script>
```

Comment:
is this needed here? can we not just add it to footer with IF S_RECAPTCHA_AVAILABLE